DESCRIPTION

```diff
@@ -37,7 +37,7 @@ Suggests:
     testthat (>= 3.0.0)
 Config/Needs/website: tidyverse/tidytemplate
 Config/testthat/edition: 3
-Config/testthat/parallel: TRUE
+Config/testthat/parallel: FALSE
```

Comment:
We explicitly _don't use_ parallel testing because we download `latest-stable` Chrome in `testthat.R` and set the `CHROMOTE_CHROME` envvar, which does not carry over into the parallel processes